modules/ROOT/pages/clauses/where.adoc

```diff
@@ -479,33 +479,36 @@ The `name` and `age` for `Andy` are returned because his name starts with 'AND'
 2+|Rows: 1
 |===
 
-
 [[query-where-patterns]]
-== Using path patterns in `WHERE`
+== Path pattern expressions
 
-[[filter-on-patterns]]
-=== Filter on patterns
+[[path-pattern-expressions]]
```

Comment:
This anchor should be placed above the header (and replace the old one (query-where-patterns), which ( checked) is currently not used).

```suggestion
```